_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -13124,6 +13732,11 @@
 /obj/structure/disposalpipe/segment,
 /obj/structure/safe/floor,
 /obj/item/reagent_containers/food/snacks/fortunecookie,
+/obj/item/a_gift/anything,
+/obj/item/a_gift/anything,
+/obj/item/a_gift/anything,
+/obj/item/a_gift/anything,
+/obj/item/a_gift/anything,
```

Comment:
You should really, really remove all of the **anything** in the **/obj/item/a_gift/anything** as those can spawn anything, from playing cards to an instakill rifle.

Reply:
Not just those I highlighted here, but the ones that are all around the station. Removing the anything part turns them in presents that only contain toys in them.